packages/react-charting/src/components/GroupedVerticalBarChart/GroupedVerticalBarChart.base.tsx

```diff
@@ -215,11 +213,8 @@ export class GroupedVerticalBarChartBase extends React.Component<
   };
 
   private _getOpacity = (legendTitle: string): string => {
-    let shouldHighlight = true;
-    if (this.state.isLegendHovered || this.state.isLegendSelected) {
-      shouldHighlight = this.state.titleForHoverCard === legendTitle;
-    }
-    return shouldHighlight ? '' : '0.1';
+    const opacity = this._legendHighlighted(legendTitle) || this._noLegendHighlighted() ? '' : '0.1';
```

Comment:
Can you confirm that this value is '' and not something else.

Reply:
The browser treats empty opacity as 100%. Instead of an empty string, '1' would be more intuitive.